### PR TITLE
set clusterset to default when value is null string

### PR DESCRIPTION
**Pull request by ldpliu**
Signed-off-by: ldpliu <daliu@redhat.com>
https://github.com/stolostron/backlog/issues/21335

### CODE REVIEW

File: pkg/webhook/cluster/mutating_webhook.go [hunk 1]
```diff
@@ -128,7 +128,19 @@ func (a *ManagedClusterMutatingAdmissionHook) addDefaultClusterSetLabel(managedC
 		return jsonPatches, status
 	}
 
-	if _, ok := managedCluster.Labels[clusterSetLabel]; ok {
+	if v, ok := managedCluster.Labels[clusterSetLabel]; ok {
```
Comment (qiujian16) [inline pkg/webhook/cluster/mutating_webhook.go]: I would do

```
v, ok := managedCluster.Labels[clusterSetLabel]
if !ok {
  // add label
}

if len(v) == 0 {
  // update label
}

return nil
```

Comment (ldpliu) [reply pkg/webhook/cluster/mutating_webhook.go]: done

**Pull request merged**